src/main/java/uk/gov/hmcts/reform/sendletter/services/ftp/SshClient.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sendletter.services.ftp;
+
+import net.schmizz.sshj.SSHClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.function.Function;
+
+/**
+ * Class to manage {@link SSHClient} with current state of service.
+ * For backwards compatibility the overridden {@link SshClient#close()}
```

Comment:
This class is a bit confusing. All it does is catching an exception when closing fails - something that should be best done outside of it, anyway (so that the caller of the tool knows).

Does backward compatibility apply if we're also changing the code that uses the SSH client?

In my opinion would be better not to have this class.

Reply:
So it was perfectly fine before? Fine